core/src/ldml/C7043_ldml.md

```diff
@@ -290,23 +270,7 @@ For each reorder item:
 
 ### C7043.2.12 `vkey`—VKey Map
 
-| ∆ | Bits | Name    | Description                              |
-|---|------|---------|------------------------------------------|
-| 0 |  32  | ident   | `vkey`                                   |
-| 4 |  32  | size    | int: Length of section                   |
-| 8 |  32  | count   | int: Number of vkeys                     |
-
-The keys are sorted in binary order based on the `vkey` field.
-
-For each key:
-
-| ∆ | Bits | Name    | Description                              |
-|---|------|---------|------------------------------------------|
-|16+|  32  | vkey    | int: source vkey ID (0…255)              |
-|20+|  32  | target  | int: target vkey ID (0…255)              |
-
-- `vkey`: Is the standard vkey, 0-255
-- `target`: Is the target (resolved) vkey, 0-255.
+_Removed._
```

Comment:
I think we'll be able to remove these references altogether given they were based on a draft spec and never released. But a later cleanup PR is appropriate

Reply:
makes sense.